### PR TITLE
add pagination to character select + fixed pagination not showing 2 button on 3 pages

### LINKED ISSUE #30: add paging to search

### PR DESCRIPTION
**Pull request by Tunnelbliick**
closes #30

### CODE REVIEW

File: src/components/generic/design/paginator.svelte [hunk 1]
```diff
@@ -18,12 +18,6 @@
 			upperBoundary--;
 		}
 
-		console.log(lowerBoundary, upperBoundary);
-
-		if (upperBoundary === lowerBoundary) {
```
Comment (lvdlee) [inline src/components/generic/design/paginator.svelte:23]: Leave this if statement in. The paginator will break with low result counts if this check isn't in there.

Comment (Tunnelbliick) [reply src/components/generic/design/paginator.svelte:23]: @lvdlee It currently skips 2 if the pagecount is 3.

I tested the change with 1-2-3-4 results and they all looked fine. Bigger results were fine aswell @

Comment (lvdlee) [reply src/components/generic/design/paginator.svelte:23]: @Tunnelbliick so it's completely tested and working like this if I understand correctly?

Comment (Tunnelbliick) [reply src/components/generic/design/paginator.svelte:23]: @lvdlee i tested it a bunch yesterday, but ill double down and test it again once im home.

Comment (Tunnelbliick) [reply src/components/generic/design/paginator.svelte:23]: image
image
image

Comment (Tunnelbliick) [reply src/components/generic/design/paginator.svelte:23]: @lvdlee high result counts move as normal aswell so i dont think this is an issue.

**Review by lvdlee — APPROVED** [review]

**Pull request merged**